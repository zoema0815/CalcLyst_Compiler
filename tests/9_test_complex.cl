int main () {
    int x;
    int y;
    int b;
    Real c;
    Real f;
    Imaginary d;
    Imaginary g;
    Complex z;
    Complex w;
    c = 3.7r;
    f = 2.5r;
    d = 3.2i;
    g = 1.42i;
    z = d~c;
    w = 5.2@2.8i;
    x = 5;
    y = 3;
    b = x / y;
    printComplex(z);
    printComplex(z/w);
    printComplex(z*w);
    printComplex(z+w);
    printComplex(z-w);
    return 0;
}