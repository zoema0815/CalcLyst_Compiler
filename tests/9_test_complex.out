3.7@3.2i
0.80848@0.18004i
10.28000@27.00000i
8.90000@6.00000i
-1.50000@0.40000i
